app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-<h2>User's posts</h2>
\ No newline at end of file
+<div class="users-container">
+  <div class="user-card">
+    <div class="user-photo">
+      <%= image_tag @user.photo, alt: "#{ @user.name }'s photo" %>
+    </div>
+    <div class="user-details">
+      <%= link_to @user.name, user_path(@user) %>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+</div>
+
+<div class="user-posts">
+  <% @posts.each do |post| %>
+    <div class="post">
+      <h3><%= link_to post.title, user_post_path(@user, post) %></h3>
+      <p><%= post.text %></p>
+      <p>Comments: <%= post.comments_counter %></p>
+      <p>Likes: <%= post.likes_counter %></p>
```

Comment:
- [ ] Great job working on the requirements so far but I think it would be great if you just add the likes and comment counters in the side as shown in the given design so please just align them as given design 🚀🔥💯💪🏻


- Your
![image](https://github.com/mryeminthu/Rails-blog/assets/58553711/328001ac-3be6-45b5-a6ba-1e9656073366)

- Given
![image](https://github.com/mryeminthu/Rails-blog/assets/58553711/207d3541-cde0-433e-92be-69c19079ecea)